tests/components/prosegur/test_camera.py

```diff
@@ -28,12 +28,9 @@ async def test_camera_fail(hass, init_integration, mock_install, caplog):
     )
 
     with caplog.at_level(logging.ERROR, logger="homeassistant.components.prosegur"):
-        try:
+        with pytest.raises(HomeAssistantError) as exc:
```

Comment:
I think we can merge the two `with` statements by using a comma between the context managers.

Reply:
right :)